src/components/LoginForm.tsx

```diff
@@ -0,0 +1,74 @@
+
+import * as React from 'react';
+import { StyleSheet, TouchableOpacity, View } from 'react-native';
+import { KeyboardAwareScrollView } from 'react-native-keyboard-aware-scroll-view';
+import CustomText from './custom-components/CustomText';
+import LoginInput from './LoginInput';
+import ProviderAuthBlock from './ProviderAuthBlock';
+
+
+const loginForm = () => {
```

Comment:
This is the functional (but still) **component**. First letter uppercase

Reply:
oops, it'll be changed